pkgs/development/python-modules/sentry-sdk/default.nix

```
@@ -1,4 +1,4 @@
-{ stdenv, buildPythonPackage, fetchPypi, urllib3, certifi, django, flask, tornado, sanic, aiohttp, bottle, rq, falcon, pyramid, celery }:
+{ stdenv, buildPythonPackage, fetchPypi, urllib3, certifi, django, flask, tornado, bottle, rq, falcon }:
```

to fix your evaluation
```suggestion
{ stdenv, buildPythonPackage, fetchPypi, isPy3k, urllib3, certifi, django, flask, tornado, bottle, rq, falcon }:
```